ert3/engine/_clean.py

```diff
@@ -0,0 +1,17 @@
+import ert3
+
+
+def clean(workspace, experiment_names, all):
+    stored_experiments = ert3.storage.get_experiment_names(workspace=workspace)
+    if all:
+        experiment_names = stored_experiments
+    else:
+        experiment_names = [
+            experiment
+            for experiment in experiment_names
+            if experiment in stored_experiments
```

Comment:
This is surprisingly forgiving. I think we should at the very least propagate some kind of warning to the user, but the easiest to implement as of now is probably to fail unless all experiment names are recognised.